### PR TITLE
Use mermaid instead of pikchr for diagrams

### PR DESCRIPTION
**Pull request by blaix**
See conversation about this that starts here:
https://github.com/gren-lang/book/pull/15#issuecomment-2126282446

Tested with https://github.com/blaix/fake-gren-package

Renders fine on the package overview and module docs pages, but the rss feed looks empty and I'm not sure why... should I have to do anything besides import a package for it to show up in the feed?

### CODE REVIEW

File: src/routes/feed.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import Router from "@koa/router";
-import pikchrPkg from "markdown-it-pikchr";
+import markdownItMermaid from '@markslides/markdown-it-mermaid';
```
Comment (robinheghan) [inline src/routes/feed.js:2]: Would you mind running `npm run prettier`?

**Comment by robinheghan** [comment]
The rss feed doesn't look empty here (looking at the raw xml output). The package site returns the first version for use in the feed (the feed is only for brand new packages), so I had to tweak the SQL statement to return the latest version for use in the feed.

It shows the raw mermaid code, but I don't think you can import modules in rss so that's the best we can do.

**Comment by blaix** [comment]
Hah, I think we were doing the same thing at the same time: https://github.com/gren-lang/package.gren-lang.org/pull/37

**Comment by robinheghan** [comment]
Is this ready to merge?

**Comment by blaix** [comment]
> Is this ready to merge?

It is now

**Pull request merged**